pkg/kubecost/allocation.go

```diff
@@ -63,6 +63,9 @@ type Allocation struct {
 	NetworkTransferBytes       float64               `json:"networkTransferBytes"`
 	NetworkReceiveBytes        float64               `json:"networkReceiveBytes"`
 	NetworkCost                float64               `json:"networkCost"`
+	NetworkZoneCost            float64               `json:"networkZoneCost"`     // @bingen:field[version=16]
+	NetworkRegionCost          float64               `json:"networkRegionCost"`   // @bingen:field[version=16]
+	NetworkInternetCost        float64               `json:"networkInternetCost"` // @bingen:field[version=16]
```

Comment:
Also -- are these versions correct? Aren't we at a higher version now?

Reply:
It is at 16 itself for allocation struct